runtime-scripts/after_startup.sh

```diff
@@ -59,13 +59,14 @@ fi
 /opt/keycloak/bin/kcadm.sh update users/profile -r open-products-facts -f /opt/off/users_profile.json
 
 # Create clients
-for CLIENT in $CLIENTS
+for CLIENT_ID in $(echo $CLIENTS | tr ',' ' ')
```

Comment:
Maybe make the list of CLIENT_IDs space delimited so we don't have to do the extra `tr`?